docs/event-bus/assets/configure-consume-events.svg

```diff
@@ -1,3 +1,3 @@
tp://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><font color="#ffffff" style="font-size: 15px"><b>3</b></font></div></div></foreignObject><text x="4" y="15" fill="#000000" text-anchor="middle" font-size="14px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><ellipse cx="647.1" cy="398.25" rx="13.950000000000001" ry="13.950000000000001" fill="#0061c2" stroke="#0061c2" stroke-width="1.8" pointer-events="none"/><ellipse cx="647.1" cy="398.25" rx="9.450000000000001" ry="9.450000000000001" fill="none" stroke="#0061c2" stroke-width="1.8" pointer-events="none"/><g transform="translate(642.5,390.5)scale(0.9)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="8" height="16" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 14px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 10px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><font color="#ffffff" style="font-size: 15px"><b>4</b></font></div></div></foreignObject><text x="4" y="15" fill="#000000" text-anchor="middle" font-size="14px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><ellipse cx="872.55" cy="522.9" rx="13.950000000000001" ry="13.950000000000001" fill="#0061c2" stroke="#0061c2" stroke-width="1.8" pointer-events="none"/><ellipse cx="872.55" cy="522.9" rx="9.450000000000001" ry="9.450000000000001" fill="none" stroke="#0061c2" stroke-width="1.8" pointer-events="none"/><g transform="translate(868.5,514.5)scale(0.9)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="8" height="16" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 14px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 10px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><font color="#ffffff" style="font-size: 15px"><b>5</b></font></div></div></foreignObject><text x="4" y="15" fill="#000000" text-anchor="middle" font-size="14px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><ellipse cx="1000.35" cy="358.65" rx="13.950000000000001" ry="13.950000000000001" fill="#0061c2" stroke="#0061c2" stroke-width="1.8" pointer-events="none"/><ellipse cx="1000.35" cy="358.65" rx="9.450000000000001" ry="9.450000000000001" fill="none" stroke="#0061c2" stroke-width="1.8" pointer-events="none"/><g transform="translate(995.5,350.5)scale(0.9)"><switch><foreignObject style="overflow:visi
```

Comment:
Natssdispatcher acts on NatssChannel CR -> that’s step 4 as well. The 4 is too far from the other line hence can be misconstrued only for NatssController